os/binfmt/libelf/libelf_init.c

```diff
@@ -185,6 +185,13 @@ int elf_init(FAR const char *filename, FAR struct elf_loadinfo_s *loadinfo)
 		return ret;
 	}
 
+#if defined(CONFIG_ELF_CACHE_READ) && !defined(CONFIG_COMPRESSED_BINARY)
+		ret = elf_cache_init(loadinfo->filfd, loadinfo->offset, &loadinfo->filelen);
+		if (ret != OK) {
+			berr("Failed to init cache support: %d\n", ret);
```

Comment:
Code alignment is not proper, also it should be #if  else for elf_cache_init and compress init.

Reply:
Done